packages/react/src/components/Dropdown/Dropdown.js

```diff
@@ -186,6 +193,7 @@ export default class Dropdown extends React.Component {
         [`${prefix}--dropdown--disabled`]: disabled,
         [`${prefix}--dropdown--light`]: light,
         [`${prefix}--dropdown--${size}`]: size,
+        [`${prefix}--dropdown--up`]: direction === 'top',
```

Comment:
```suggestion
        [`${prefix}--list-box--up`]: direction === 'top'
```
I think this might be why Dropdown isn't working?